source/_integrations/remootio.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: Remootio
+ha_release: ???
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_domain: remootio
+ha_codeowners:
+  - '@ivgg-me'
+ha_config_flow: true
```

Comment:
missing `description` and `ha_platforms`

Reply:
I have created my documentation according the [documentation of Home Assistant](https://developers.home-assistant.io/docs/documenting/create-page). Is this part of the Home Assistant documentation out of date?

Reply:
Fixed.

Reply:
it's a little bit out of date 😉